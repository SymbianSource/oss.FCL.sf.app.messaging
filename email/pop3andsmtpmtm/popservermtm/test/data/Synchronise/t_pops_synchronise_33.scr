+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted
+OK 20 30509
+OK 20 messages
1 1519
2 1558
3 1599
4 1559
5 1355
6 1519
7 1519
8 1598
9 1557
10 1598
11 1519
12 1599
13 1597
14 1558
15 1312
16 1476
17 1477
18 1557
19 1516
20 1517
.
+OK 20 messages
1 t_pops_synchronise:message026
2 t_pops_synchronise:message027
3 t_pops_synchronise:message028
4 t_pops_synchronise:message029
5 t_pops_synchronise:message030
6 t_pops_synchronise:message031
7 t_pops_synchronise:message032
8 t_pops_synchronise:message033
9 t_pops_synchronise:message034
10 t_pops_synchronise:message035
11 t_pops_synchronise:message036
12 t_pops_synchronise:message037
13 t_pops_synchronise:message038
14 t_pops_synchronise:message039
15 t_pops_synchronise:message040
16 t_pops_synchronise:message041
17 t_pops_synchronise:message042
18 t_pops_synchronise:message043
19 t_pops_synchronise:message044
20 t_pops_synchronise:message045
.
-ERR Unknown Command "CAPA"
+OK TOP 20 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message045; Wed, 15 Aug 2001 09:50:33 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2011872; 15 Aug 2001 9:43 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010255623a20cc@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:43:16 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4V>; Wed, 15 Aug 2001 10:44:57 +0100
Message-ID: <t_pops_synchronise_message045@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 45
Date: Wed, 15 Aug 2001 10:37:06 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty-Five Forty-Five Forty-Five Forty-Five Forty-Five Forty-Five Forty-Five
.
+OK TOP 19 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message044; Wed, 15 Aug 2001 09:42:41 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ab1104200; 15 Aug 2001 9:41 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102556239d982@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:42:58 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB44>; Wed, 15 Aug 2001 10:44:41 +0100
Message-ID: <t_pops_synchronise_message044@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 44
Date: Wed, 15 Aug 2001 10:36:49 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty-Four Forty-Four Forty-Four Forty-Four Forty-Four Forty-Four Forty-Four
.
+OK TOP 18 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message043; Wed, 15 Aug 2001 09:48:13 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ac1104200; 15 Aug 2001 9:41 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010255623991da@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:42:40 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4S>; Wed, 15 Aug 2001 10:44:27 +0100
Message-ID: <t_pops_synchronise_message043@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 43
Date: Wed, 15 Aug 2001 10:36:34 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty-Three Forty-Three Forty-Three Forty-Three Forty-Three Forty-Three
.
+OK TOP 17 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message042; Wed, 15 Aug 2001 09:43:52 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1104200; 15 Aug 2001 9:41 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010255623904bd@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:42:04 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4Q>; Wed, 15 Aug 2001 10:44:07 +0100
Message-ID: <t_pops_synchronise_message042@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 42
Date: Wed, 15 Aug 2001 10:36:16 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty-Two Forty-Two Forty-Two Forty-Two Forty-Two Forty-Two Forty-Two
.
+OK TOP 16 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message041; Wed, 15 Aug 2001 09:41:00 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2124983; 15 Aug 2001 9:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac1001025562387687@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:41:27 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4N>; Wed, 15 Aug 2001 10:43:45 +0100
Message-ID: <t_pops_synchronise_message041@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 41
Date: Wed, 15 Aug 2001 10:35:54 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty-One Forty-One Forty-One Forty-One Forty-One Forty-One Forty-One
.
+OK TOP 15 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message040; Wed, 15 Aug 2001 09:45:06 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1007624; 15 Aug 2001 9:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac1001025562382692@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:41:07 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4M>; Wed, 15 Aug 2001 10:43:30 +0100
Message-ID: <t_pops_synchronise_message040@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 40
Date: Wed, 15 Aug 2001 10:35:39 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Forty Forty Forty Forty Forty Forty Forty Forty Forty Forty 
.
+OK TOP 14 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message039; Wed, 15 Aug 2001 09:56:39 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ab1007624; 15 Aug 2001 9:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac1001025562375289@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:40:12 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB42>; Wed, 15 Aug 2001 10:43:09 +0100
Message-ID: <t_pops_synchronise_message039@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 39
Date: Wed, 15 Aug 2001 10:35:18 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Nine Thirty-Nine Thirty-Nine Thirty-Nine Thirty-Nine Thirty-Nine
.
+OK TOP 13 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message038; Wed, 15 Aug 2001 09:40:47 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1127261; 15 Aug 2001 9:38 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102556236c04c@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Wed, 15 Aug 2001 10:39:35 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHB4G>; Wed, 15 Aug 2001 10:42:47 +0100
Message-ID: <t_pops_synchronise_message038@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 38
Date: Wed, 15 Aug 2001 10:34:55 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Eight Thirty-Eight Thirty-Eight Thirty-Eight Thirty-Eight
.
** Read cancelled
+OK RETR 12 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message037; Tue, 14 Aug 2001 16:43:16 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aj2023324; 14 Aug 2001 15:48 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e630080@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 16:49:26 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHALV>; Tue, 14 Aug 2001 16:51:47 +0100
Message-ID: <t_pops_synchronise_message037@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 37
Date: Tue, 14 Aug 2001 16:43:59 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven
Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven 
Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven
Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven 
Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven
Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven 
Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven
Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven Thirty-Seven 
.
** Read cancelled
+OK disconnecting
