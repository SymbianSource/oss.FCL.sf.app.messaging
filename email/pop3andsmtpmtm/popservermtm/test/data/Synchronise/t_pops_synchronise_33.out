STAT
LIST
UIDL
CAPA
TOP 20 1
TOP 19 1
TOP 18 1
TOP 17 1
TOP 16 1
TOP 15 1
TOP 14 1
TOP 13 1
RETR 12
QUIT
